components/mdc/Checkbox/Checkbox.svelte

```diff
@@ -47,3 +49,15 @@ const handleChange = () => dispatch(checkbox.checked ? 'checked' : 'unchecked')
   </div>
   <label class:uppercase for={inputID}>{label}</label>
 </div>
+<div class="mdc-text-field-helper-line" style="margin-left: 10px">
```

Comment:
```suggestion
<div class="mdc-text-field-helper-line ml-10px">
```
We have a utility class for that

Reply:
Awesome, thanks for pointing that out